Client/Frontend/Home/ActivityStreamTopSitesCell.swift

```diff
@@ -242,6 +254,20 @@ class ASHorizontalScrollCell: UITableViewCell {
         pageControl.progress = currentPage
     }
 
+    func handlePageTap(gesture: UITapGestureRecognizer) {
+        if pageControl.pageCount <= 1 {
```

Comment:
Nit: Would read better as a `guard` since we're using the if as a guard to the code below.